Assets/MixedRealityToolkit.Extensions/Providers/Experimental/UnityAR/UnityARCameraSettings.cs

```diff
@@ -188,9 +189,12 @@ private void InitializeARFoundation()
             if (SettingsProfile != null)
             {
                 // Read settings to be applied to the camera.
-                poseSource = SettingsProfile.PoseSource;
-                trackingType = SettingsProfile.TrackingType;
-                updateType = SettingsProfile.UpdateType;
+                // The enums used by the profile use the same values as those provided by Unity.
+                // We use custom enums to avoid customers needing to include packages and/or namespaces simply to resolve
+                // enum values in the profile.
+                poseSource = (TrackedPoseDriver.TrackedPose)SettingsProfile.PoseSource;
```

Comment:
Do we want to run switches here, in case the order / numbering of the Unity enums change?

Reply:
we could